testdata/baselines/reference/fourslash/findAllReferences/findAllRefsTypedef_importType.baseline.jsonc

```diff
@@ -12,10 +12,6 @@
 // /** @typedef {number} /*FIND ALL REFS*/[|Foo|] */
 // const dummy = 0;
 
-// === /b.js ===
-// /** @type {import('./a').[|Foo|]} */
-// const x = 0;
-
```

Comment:
This seems like a regression, maybe? `a.js` theoretically is a module because it has `module.exports` in it, I think, but now we are losing that export and therefore the reference.

Reply:
Yeah, that's an oversight. We need to handle the implied export of `@typedef` in find-all-references.